src/inc/conpty-static.h

```diff
@@ -11,6 +11,18 @@
 
 #include <consoleapi.h>
 
+#ifndef CONPTY_IMPEXP
+#define CONPTY_IMPEXP __declspec(dllimport)
+#endif
+
+#ifndef CONPTY_EXPORT
+#ifdef __cplusplus
+#define CONPTY_EXPORT extern "C" CONPTY_IMPEXP
+#else
+#define CONPTY_EXPORT extern CONPTY_IMPEXP
+#endif

```

Comment:
This isn't necessary. You're already wrapping the entire file in a `extern "C"`.
In other words, you can use `CONPTY_IMPEXP` directly.